tiny-require.js

```diff
@@ -0,0 +1,53 @@
+// Native implementation of resource loader based on jQuery ajax
+
+(function(global) {
+
+  // cache storage for resources sucsessfully loaded 
+  var _resources = {};
+  
+  var tinyRequire = function(paths, callback) {
```

Comment:
Implemented resource loader based on jQuery ajax. It replaces the previous dependencies solution, which was not as readable and manageable.